x-pack/plugins/ml/public/application/datavisualizer/index_based/components/field_data_card/content_types/date_content.tsx

```diff
@@ -4,64 +4,77 @@
 
+import { i18n } from '@kbn/i18n';
 import { FieldDataCardProps } from '../field_data_card';
-import { roundToDecimalPlace } from '../../../../../formatters/round_to_decimal_place';
-
+import { ExpandedRowFieldHeader } from '../../../../stats_datagrid/components/expanded_row_field_header';
 const TIME_FORMAT = 'MMM D YYYY, HH:mm:ss.SSS';
+interface SummaryTableItem {
+  function: string;
+  display: ReactNode;
+  value: number | string | undefined | null;
+}
 
 export const DateContent: FC<FieldDataCardProps> = ({ config }) => {
   const { stats } = config;
+  if (stats === undefined) return null;
 
-  const { count, sampleCount, earliest, latest } = stats;
-  const docsPercent = roundToDecimalPlace((count / sampleCount) * 100);
-
-  return (
-    <div className="mlFieldDataCard__stats">
-      <div>
-        <EuiText size="xs" color="subdued" data-test-subj="mlFieldDataCardDocCount">
-          <EuiIcon type="document" />
-          &nbsp;
-          <FormattedMessage
-            id="xpack.ml.fieldDataCard.cardDate.documentsCountDescription"
-            defaultMessage="{count, plural, zero {# document} one {# document} other {# documents}} ({docsPercent}%)"
-            values={{
-              count,
-              docsPercent,
-            }}
-          />
-        </EuiText>
-      </div>
-
-      <EuiSpacer size="m" />
+  const { earliest, latest } = stats;
 
-      <div data-test-subj="mlFieldDataCardEarliest">
+  const summaryTableTitle = i18n.translate('xpack.ml.fieldDataCard.cardDate.summaryTableTitle', {
+    defaultMessage: 'Summary',
+  });
+  const summaryTableItems = [
+    {
+      function: 'earliest',
+      display: (
+        <FormattedMessage id="xpack.ml.fieldDataCard.cardDate.minLabel" defaultMessage="min" />
+      ),
+      value: formatDate(earliest, TIME_FORMAT),
+    },
+    {
+      function: 'latest',
+      display: (
         <FormattedMessage
-          id="xpack.ml.fieldDataCard.cardDate.earliestDescription"
-          defaultMessage="earliest {earliestFormatted}"
-          values={{
-            earliestFormatted: formatDate(earliest, TIME_FORMAT),
-          }}
+          id="xpack.ml.fieldDataCard.cardDate.medianLabel"
+          defaultMessage="median"
         />
-      </div>
-
-      <EuiSpacer size="s" />
+      ),
+      value: formatDate(latest, TIME_FORMAT),
+    },
+  ];
+  const summaryTableColumns = [
+    {
+      name: '',
+      render: (summaryItem: { display: ReactNode }) => summaryItem.display,
+      width: '75px',
+    },
+    {
+      field: 'value',
+      name: '',
+      render: (v: string) => <strong>{v}</strong>,
+    },
+  ];
 
-      <div data-test-subj="mlFieldDataCardLatest">
-        <FormattedMessage
-          id="xpack.ml.fieldDataCard.cardDate.latestDescription"
-          defaultMessage="latest {latestFormatted}"
-          values={{
-            latestFormatted: formatDate(latest, TIME_FORMAT),
-          }}
+  return (
+    <EuiFlexGroup direction={'row'}>
```

Comment:
Can you explain why we need a flex layout with a single item here?

Reply:
Updated here https://github.com/elastic/kibana/pull/85726/commits/0dcdf2efbdbfdbd6a0259f38cc37bac7942cfae7